src/dataservice/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
```

Comment:
You also need to update the pom under src to also build the dataservice module (also during the CI build)